thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/rootcause-table.ftl

```diff
@@ -1,4 +1,22 @@
 <div class="analysis-card padding-all top-buffer">
   <h4 class="analysis-title bottom-buffer">[BETA] Root Cause Search</h4>
-  <table id="rootcause-data-table" class="table table-borderless rootcause__table"></table>
+  <ul class="nav nav-tabs" role="tablist">
+    <li><a class="active" data-toggle="tab" data-selector="">All</a></li>
+    <li><a data-toggle="tab" data-selector="SERVICE">Services</a></li>
+    <li><a data-toggle="tab" data-selector="HOLIDAY">Holidays</a></li>
+  </ul>
+  <div class="tab-content">
+    <div class="tab-pane active" id="tab-table1">
+      <table id="rootcause-data-table" class="table table-borderless rootcause__table"></table>
+    </div>
+  </div>
 </div>
+
+<script type="application/javascript">
```

Comment:
Can we move this inline javascript script into a Js file and perhaps call it after the page is rendered?